app/src/main/kotlin/com/jakewharton/sa4p/ui/footer.kt

```diff
@@ -34,8 +34,11 @@ import com.jakewharton.sa4p.presenter.Authenticated
 import com.jakewharton.sa4p.presenter.MainModel
 import com.jakewharton.sa4p.presenter.Unauthenticated
 
+@Suppress("ktlint:compose:modifier-missing-check") // BottomAppBar has its own modifier
 @Composable
-fun BottomBar(model: MainModel) {
+fun BottomBar(
+	model: MainModel,
+) {
 	BottomAppBar(
 		modifier = Modifier.wrapContentHeight(),
```

Comment:
I still think this should take a parameter, but instead you use it as the initial modifier value.
```suggestion
		modifier = modifier.wrapContentHeight(),
```

Reply:
I recalled that applying this would conflict with another rule, but it works fine now.